feedback-front/templates/add_record.html

```diff
@@ -13,6 +13,12 @@
   {% endif %}
   {% if success %}<div class="uk-alert">{{ success }}</div>{% endif %}
 
+  <ul class="uk-breadcrumb">
+    <li><a href="http://localhost:5000">Home</a></li>
+    <li><a href="http://localhost:5000">{{dataset}}</a></li>
```

Comment:
No need to add `http://localhost:5000` - just `/` is fine.
And what is `<li><a href="http://localhost:5000">{{dataset}}</a></li>` doing? Leading back to home? I think you mean `<li><a href="/{{dataset}}">{{dataset}}</a></li>`